### PR TITLE
Add new NeutronRC AIO targets

### PR DESCRIPTION
**Pull request by DusKing1**
All are AIO boards with ESCs, so motor config is safe in this case.

### CODE REVIEW

File: configs/default/NERC-NEUTRONRCH743AIO.config [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+manufacturer_id NERC
+
+# resources
+resource BEEPER 1 A15
+resource MOTOR 1 B00
+resource MOTOR 2 B01
+resource MOTOR 3 A00
+resource MOTOR 4 A01
+resource MOTOR 5 A02
+resource MOTOR 6 A03
```
Comment (haslinghuis) [inline configs/default/NERC-NEUTRONRCH743AIO.config:13]: Should we use a single timer using resource MOTOR A01-A03 for MOTOR 1-4?

Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH743AIO.config:13]: why can't?

Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH743AIO.config:13]: A00, A01, A02, A03 are  for motor 3, 4, 5, 6

Comment (haslinghuis) [reply configs/default/NERC-NEUTRONRCH743AIO.config:13]: Just would like to have motor 1-4 on a sinlge DMA and single pace timer.

Currently:
```
resource MOTOR 1 B00 timer B00 AF2 # pin B00: TIM3 CH3 (AF2) dma pin B00 0 # pin B00: DMA1 Stream 0 Request 25
resource MOTOR 2 B01 timer B01 AF2 # pin B01: TIM3 CH4 (AF2) dma pin B01 1 # pin B01: DMA1 Stream 1 Request 26
resource MOTOR 3 A00 timer A00 AF2 # pin A00: TIM5 CH1 (AF2) dma pin A00 2 # pin A00: DMA1 Stream 2 Request 55
resource MOTOR 4 A01 timer A01 AF2 # pin A01: TIM5 CH2 (AF2) dma pin A01 3 # pin A01: DMA1 Stream 3 Request 56
resource MOTOR 5 A02 timer A02 AF2 # pin A02: TIM5 CH3 (AF2) dma pin A02 4 # pin A02: DMA1 Stream 4 Request 57
resource MOTOR 6 A03 timer A03 AF2 # pin A03: TIM5 CH4 (AF2) dma pin A03 5 # pin A03: DMA1 Stream 5 Request 58
resource MOTOR 7 D12 timer D12 AF2 # pin D12: TIM4 CH1 (AF2) dma pin D12 6 # pin D12: DMA1 Stream 6 Request 29
resource MOTOR 8 D13 timer D13 AF2 # pin D13: TIM4 CH2 (AF2) dma pin D13 7 # pin D13: DMA1 Stream 7 Request 30
```

Requesting:
```
resource MOTOR 1 A00 timer A00 AF2 # pin A00: TIM5 CH1 (AF2) dma pin A00 2 # pin A00: DMA1 Stream 2 Request 55
resource MOTOR 2 A01 timer A01 AF2 # pin A01: TIM5 CH2 (AF2) dma pin A01 3 # pin A01: DMA1 Stream 3 Request 56
resource MOTOR 3 A02 timer A02 AF2 # pin A02: TIM5 CH3 (AF2) dma pin A02 4 # pin A02: DMA1 Stream 4 Request 57
resource MOTOR 4 A03 timer A03 AF2 # pin A03: TIM5 CH4 (AF2) dma pin A03 5 # pin A03: DMA1 Stream 5 Request 58
resource MOTOR 5 B00 timer B00 AF2 # pin B00: TIM3 CH3 (AF2) dma pin B00 0 # pin B00: DMA1 Stream 0 Request 25
resource MOTOR 6 B01 timer B01 AF2 # pin B01: TIM3 CH4 (AF2) dma pin B01 1 # pin B01: DMA1 Stream 1 Request 26
resource MOTOR 7 D12 timer D12 AF2 # pin D12: TIM4 CH1 (AF2) dma pin D12 6 # pin D12: DMA1 Stream 6 Request 29
resource MOTOR 8 D13 timer D13 AF2 # pin D13: TIM4 CH2 (AF2) dma pin D13 7 # pin D13: DMA1 Stream 7 Request 30
```

Comment (DusKing1) [reply configs/default/NERC-NEUTRONRCH743AIO.config:13]: This couldn't be done as the hardware already being produced.

**Review by haslinghuis — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Pull request merged**